feed/models.py

```diff
@@ -0,0 +1,61 @@
+from django.db import models
+from django.core.exceptions import ValidationError
+from django.db.models import Q
+
+from django.contrib.auth.models import User
+from course.models import TeacherCourses, StudentCourses
+
+
+class PostManager(models.Manager):
```

Comment:
I'm not against this class, but is it needed? what are the future plans for it? because I think `get_posts...` can be done in a class method for `Post`

Reply:
I decided to separate the `get_posts...` method from the Post class and use a manager instead because:
1. This method is a "table level" query and not tied to a specific instance of Post

2. Relatively complex data operation compared to other queries inside the class methods.
Inside course docs for demo 2, it was defined that we need managers for:
`Manager classes for the more complex data operations you expect to do in your applications`